test/service/execution.test.ts

```diff
@@ -0,0 +1,61 @@
+import { expect } from 'chai';
+import { it, describe } from 'mocha';
+
+import { timeout } from '../../src/util/promise';
+import SyncConfig from '../../src/domain/SyncConfig';
+import * as executionService from '../../src/services/execution';
+
+const getProcesses = (tracker: string[]) => [
+  () =>
+    timeout(45)
```

Comment:
Add a test where the promiser rejects something?

Reply:
Done.